crates/bevy_core_pipeline/src/lib.rs

```diff
@@ -9,6 +9,8 @@ pub mod prelude {
     pub use crate::ClearColor;
 }
 
+use std::collections::HashMap;
```

Comment:
There's a bevy internal HashMap and I'm not sure when we're supposed to use that versus std:collections::HashMap or if we're always supposed to use the bevy type to make it easy to change later... I think it defaults to ahash or something, which is faster than the std::collections default I think.

Reply:
Yep, we should be using `bevy::utils::HashMap` here. We should always use it, unless we care about cryptographic security.

Reply:
Done.